tests/rptest/direct_consumer_tests/direct_consumer_test.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2025 Redpanda Data, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+from typing import Any
+from rptest.context.cloud_storage import TestContext
+from rptest.services.cluster import cluster
+from ducktape.utils.util import wait_until
+
+from rptest.services.direct_consumer_verifier import (
+    DirectConsumerVerifier, CreateDirectConsumerRequest, BrokerAddress,
+    DirectConsumerConfiguration, OffsetResetPolicy, IsolationLevel,
+    AssignPartitionsRequest, TopicAssignment, PartitionAssignment,
+    GetConsumerStateRequest)
+from rptest.services.kgo_verifier_services import KgoVerifierProducer
+from rptest.clients.types import TopicSpec
+from rptest.services.redpanda import SISettings
+from rptest.tests.redpanda_test import RedpandaTest
+
+
+#TODO: This test must be enabled once the direct consumer verifier support
+# is added to vtools.
+class DirectConsumerVerifierTest(RedpandaTest):
```

Comment:
is it disabled now? I don't see any `@ignore`, so how do we turn it on when ready?

Reply:
It is in a separate directory i.e. it is not included in any of the test suites